goalFormulas.gs

```diff
@@ -22,11 +22,8 @@ function formatRawDataForExport(rawDataArray){
     if(i == 0){
       emptyRow = allEntries.map(_ => `""`);
     }
-    const rowIsNotEmpty = allEntries.filter(n => n != '\"\"').join('') != '';
-    return rowIsNotEmpty
-      ? `[${allEntries.join(',')}],`
-      : null;
-  }).filter(n => n).concat(`[${emptyRow.join(',')}]`);
+    return `[${allEntries.join(',')}],`
+  }).filter(n => getCamelCase(n) != '').concat(`[${emptyRow.join(',')}]`);
```

Comment:
getCamelCase() takes a string and removes all the special characters so by a happy accident it can take the array.ToString()  to see if it is empty and filter it out.